client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       type: string
     - description: Determines whether a VolumeSnapshotContent created through the
         VolumeSnapshotClass should be deleted when its bound VolumeSnapshot is deleted.
-      jsonPath: .deletionPolicy
+    - jsonPath: .deletionPolicy
```

Comment:
This file is generated automatically.  We usually don't manually edit like this.